client/configuration_set.go

```diff
@@ -0,0 +1,101 @@
+package client
+
+type CreateConfigurationSetPayload struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	// if Scope is "organization", scopeId will be calculated in the functions.
+	Scope                   string                  `json:"scope"`   // "project" or "organization".
+	ScopeId                 string                  `json:"scopeId"` // project id or organization id.
+	ConfigurationProperties []ConfigurationVariable `json:"configurationProperties"`
+}
+
+type UpdateConfigurationSetPayload struct {
+	Name                           string                  `json:"name"`
+	Description                    string                  `json:"description"`
+	ConfigurationPropertiesChanges []ConfigurationVariable `json:"configurationPropertiesChanges"` // delta changes.
+}
+
+type ConfigurationSet struct {
+	Id string `json:"id"`
```

Comment:
the return also contain the `name` and the `description`

Reply:
yes I know. Not really needed. But I'll add it.